web/src/lib/components/Dashboard/Map/Map.svelte

```diff
@@ -132,9 +143,77 @@
 			);
 		}, 100);
 
-		viewer.trackedEntity = circlingPoint;
+		//viewer.trackedEntity = circlingPoint;
+
+		var test = viewer.entities.add({
+			position: ottawaPosition,
+			model: {
+				uri: "../../../../../static/models/rocket.glb",
+				runAnimations: false,
+				scale: 1.0, // Optional: Adjust the scale of the model if needed
+       			minimumPixelSize: 50 // Optional: Minimum pixel size for the model*/
+			},
+			label: {
+				text: 'Altitude: 1000 meters', // Initial altitude label
+				font: '14px sans-serif',
+				fillColor: Cesium.Color.YELLOW,
+				outlineColor: Cesium.Color.BLACK,
+				outlineWidth: 2,
+				style: Cesium.LabelStyle.FILL_AND_OUTLINE,
+				pixelOffset: new Cesium.Cartesian2(0, -100), // Offset label position
+				//heightReference: Cesium.HeightReference.CLAMP_TO_GROUND, // Altitude relative to ground
+				backgroundColor: Cesium.Color.WHITE
+			},
+				/*point: {
+				pixelSize: 10,
+				color: Cesium.Color.BLACK
+			},*/
+
+		})
+
+		viewer.flyTo(test);
+
+		// Subscribe to altitude changes and update label text
+		/*LatestAltitudeMeasurements.subscribe((data: any) => {
+			console.log('data:', data);
+			var altitude = data.data.rocket_sensor_air[0].altitude;
+			console.log('Altitude:', altitude);
+
+	
+			if (test.label) {
+				test.label.text = new Cesium.ConstantProperty(`Altitude: ${altitude} meters`);
+				const zPosition = altitude * 1000; //adjust scale factor etc
+				test.position = new Cesium.ConstantPositionProperty(
+					Cesium.Cartesian3.fromDegrees(-75.69, 45.42, zPosition)
+				);
+			}
+		});*/
+
+		LatestCoordinates.subscribe((data: any) => { 
+			console.log('data:', data);
+			var latitude = data.data.rocket_sensor_gps_pos_1[0].latitude;
+			var longitude = data.data.rocket_sensor_gps_pos_1[0].longitude;
+			var altitude = data.data.rocket_sensor_gps_pos_1[0].altitude;
+
+			console.log('Altitude:', altitude);
+			console.log('Latitude:', latitude);
+			console.log('Longitude:', longitude);
+
+	
+			if (test.label) {
+				test.label.text = new Cesium.ConstantProperty(`Latitude: ${latitude}, Longitude: ${longitude}, Altitude: ${altitude} meters`);
+				const zPosition = altitude * 1000; //adjust scale factor etc
+				test.position = new Cesium.ConstantPositionProperty(
+				Cesium.Cartesian3.fromDegrees(longitude, latitude, zPosition)
+				);
+			}
+		});
```

Comment:
This looks alright so far. 

Can you use reactive statements like in RocketNavBall?

![image](https://github.com/uorocketry/rgs/assets/26195439/64da68b7-7917-46d2-9afb-ae916c896d4b)

this avoids explicitly calling subscribe (which needs to be unsubscribed) the other way unsubscribes the data automatically.

Reply:
can I unsubscribe using svelte unsubscribe or do you want it to be a reactive statement ?